api-admin/src/main/java/bio/terra/pearl/api/admin/service/notifications/SendgridActivityService.java

```diff
@@ -0,0 +1,70 @@
+package bio.terra.pearl.api.admin.service.notifications;
+
+import bio.terra.pearl.core.dao.notification.SendgridEventDao;
+import bio.terra.pearl.core.model.notification.SendgridEvent;
+import bio.terra.pearl.core.service.notification.email.SendgridClient;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+@Service
+public class SendgridActivityService {
+  private static final Logger logger = LoggerFactory.getLogger(SendgridActivityService.class);
```

Comment:
Dennis just clued me in that we can use a lombok @Slf4j annotation and that will automatically add an appropriate logger for the class.  I'm very excited about that because I always copy-paste these loggerFactory statements and then forget to change the classname :P